js/ui/tab_cookies.js

```diff
@@ -367,23 +419,88 @@
       texts: {
         name: cookie.name,
         value: cookie.value
+      },
+      titles: {
+         whitelist: getMsg("whitelistSublistCookie")
```

Comment:
As mentioned in the solution, I was hoping we can get the message from the cache, rather than relying on the Browser API to cache current request.

Reply:
So basically we just need to declare `getMsg("whitelistSublistCookie")` outside of current function. Having that in top of the file should be enough smth like:

```
const cookieWhitelistButtonTitle = getMsg("whitelistSublistCookie");
const domainWhitelistButtonTitle = ...
```